### PR TITLE
feat: Make use of db:reset in all test suites (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**

### CODE REVIEW

File: cypress/e2e/0-smoke.cy.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const lastName = randLastName();
 describe('Authentication', () => {
 	beforeEach(() => {
 		cy.task('db:reset');
+		Cypress.session.clearAllSavedSessions();
```
Comment (netroy) [inline cypress/e2e/0-smoke.cy.ts:12]: We should instead change [`db:reset`](https://github.com/n8n-io/n8n/blob/master/cypress.config.js#L16) to 
```
				'db:reset': () =>
					Promise.all([
						fetch(BASE_URL + '/e2e/db/reset', { method: 'POST' }),
						Cypress.session.clearAllSavedSessions(),
					]),
```

Comment (alexgrozav) [reply cypress/e2e/0-smoke.cy.ts]: Unfortunately cannot use `Cypress.session` outside of test suites. I've renamed the tasks though.

Comment (netroy) [reply cypress/e2e/0-smoke.cy.ts]: let's merge this as it for now.

**Comment by netroy** [comment]
@alexgrozav are the tests all passing for you? I'm getting this 
image

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -120,3 +119,12 @@ Cypress.Commands.add('skipSetup', () => {
 		});
 	});
 })
+
+Cypress.Commands.add('resetDatabase', () => {
```
Comment (netroy) [inline cypress/support/commands.ts:123]: since this is resetting the db as well as the session, should we rename this to just `reset` ?

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.205.0`

**Pull request merged**